ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -367,14 +367,18 @@ private final class SqlLedger(
               reason,
           ),
           _ => {
+            val divulgedContracts = Nil
+            val blindingInfo = None // Currently derived from Fetch and LookupByKey nodes, soon pre-computed
```

Comment:
That's not the case for this implementation. This is effectively the committer in the Sandbox Classic style integration. The `SubmittedTransaction` contains `Fetch` and `LookupByKey` nodes as it comes directly from the DAML Engine.
We could remove these nodes as a pre-commit step after computing and supplying the blinding info, but it would still be contained within this class.

Reply:
Right. I think it's just simpler to avoid implementing this storage reduction altogether in this unified-ledger-and-indexer case, as we don't really need it here. Ty, I'll fix the comment in the next PRs.